modules/installation-and-upgrade/pages/connect-payg.adoc

```diff
@@ -7,13 +7,13 @@ In the three major public cloud providers (AWS, GCP and Azure), SUSE:
 * operates per-region RMT Servers mirroring repositories for products available as {payg}
 
 This document describes how to connect existing {payg} instance to {productname} server, and gives basic information about credentials collection from the instance.
-The goal of the connection is to extract authentication data so {productname} server is able to connect to cloud RMT host.
+The goal of the connection is to extract authentication data so {productname} server is able to connect to cloud RMT host. That will give the {productname} server access to products that are on the RMT host and that are not already available via the {scc} organization credentials.
```

Comment:
The goal of this connection is to extract authentication data so the {productname} server can connect to a cloud RMT host. {productname} server then has access to products on the RMT host which are not available via {scc} organization credentials.

Reply:
Yes, please remind I'm not a native speaker.

But maybe keep "the" in front of "{scc} organization credentials" ? we are speaking of specific credentials, the ones that are already entered ; not any credentials in general.